tmt/base.py

```diff
@@ -86,6 +86,12 @@ def __init__(self, node, parent=None):
         # Check whether the node is enabled, handle the default
         self._check('enabled', expected=bool, default=True)
 
+        # Check whether the order is integer, handle the default
+        self._check('order', expected=int, default=0)
+        if self.order < 0:
```

Comment:
We've discussed this on today's hacking session and the conclusion was that negative numbers should be allowed as well to provide enough space if the number of objects is large.